libs/vkd3d/meson.build

```diff
@@ -20,6 +20,7 @@ vkd3d_shaders =[
 ]
 
 vkd3d_src = [
+  'cache.c',
```

Comment:
Makefile.am as well (I look forward to when we nix autotools).

Reply:
Fixed.